docs/content/commands/npm-audit.md

```diff
@@ -190,5 +190,4 @@ $ npm audit --audit-level=moderate
 ### See Also
 
 * [npm install](/commands/npm-install)
-* [package-locks](/configuring-npm/package-locks)
```

Comment:
It just felt somewhat out of place, especially in the "configuring-npm" section, since it's much more conceptual.

Maybe it should be brought back in (and updated for correctness) in the using-npm section?

Also, maybe Arborist.audit() should do `this.loadVirtual().catch(() => this.loadActual())` or `.catch(() => this.buildIdealTree())`, so a lockfile _isn't_ required? 🤔

Reply:
Screw the docs, yes please do that :-D